pymc/model.py

```diff
@@ -1001,23 +1001,15 @@ def cont_vars(self):
 
     @property
     def test_point(self) -> Dict[str, np.ndarray]:
-        """Deprecated alias for `Model.compute_initial_point(seed=None)`."""
+        """Deprecated alias for `Model.initial_point(seed=None)`."""
         warnings.warn(
-            "`Model.test_point` has been deprecated. Use `Model.compute_initial_point(seed=None)`.",
+            "`Model.test_point` has been deprecated. Use `Model.initial_point(seed=None)`.",
             FutureWarning,
         )
-        return self.compute_initial_point()
+        return self.initial_point()
 
     @property
-    def initial_point(self) -> Dict[str, np.ndarray]:
-        """Deprecated alias for `Model.compute_initial_point(seed=None)`."""
-        warnings.warn(
-            "`Model.initial_point` has been deprecated. Use `Model.compute_initial_point(seed=None)`.",
-            FutureWarning,
-        )
-        return self.compute_initial_point()
-
-    def compute_initial_point(self, seed=None) -> Dict[str, np.ndarray]:
+    def initial_point(self, seed=None) -> Dict[str, np.ndarray]:
```

Comment:
The tests are failing because you made this a property and not method. Properties do not accept arguments

Reply:
More generally you should try and run the code locally as well as some of the related tests to check your changes are behaving as expected. 

This can and should be done before pushing your commits to github.

Reply:
Sure, I'll remember that.